src/wutil.cpp

```diff
@@ -564,7 +564,7 @@ file_id_t file_id_t::file_id_from_stat(const struct stat *buf)
 #if defined(__APPLE__) || defined(__DragonFly__) || defined(__FreeBSD__) ||  defined(__OpenBSD__) || defined(__NetBSD__)
     result.generation = buf->st_gen;
 #else
-    result.generation = 0;
+    result.generation = NULL;
```

Comment:
I don't care to pay to access the actual standards as written, I use basically whatever Dash on OS X has on hand for most languages. For C++ I have [this resource pop up if I type in NULL](http://en.cppreference.com/w/cpp/types/NULL) which aggregates information on how the standard has evolved over time.